src/pages/pt-BR/integrations/integrations.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: ~/layouts/MainLayout.astro
+title: Feito com Astro
+i18nReady: true
+---
+
+Membros da comunidade Astro tem com sucesso integrado serviços de terceiros em seus websites Astro!
```

Comment:
same here, I think this make more sense in english but feels unnatural in portuguese:
```suggestion
Membros da comunidade Astro tem integrado com sucesso os serviços de terceiros em seus websites Astro!
```

Reply:
I agree, thanks!